include/types.hpp

```diff
@@ -130,5 +130,19 @@ using DbusInvalidArgument =
 using InvalidArgument = phosphor::logging::xyz::openbmc_project::Common::InvalidArgument;
 
 namespace DeviceError = sdbusplus::xyz::openbmc_project::Common::Device::Error;
+
+using EventAdditionalData = std::unordered_map<std::string, std::string>;
```

Comment:
Meaningful name which can set the context of type;
eg: InventoryCallOutData.

Reply:
updated